eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/model/dto/ProfileDTO.java

```diff
@@ -13,11 +13,11 @@
 @NoArgsConstructor
 public class ProfileDTO {
```

Comment:
Ennek az osztálynak a mezőit lásd el a hibernate validator annotációval. Aztán a Controllerben kell egy `@Valid` a paraméter elé